app/src/services/audio.ts

```diff
@@ -1,3 +1,8 @@
-const AudioService = () => {}
+import TrackPlayer, { Event } from "react-native-track-player"
 
-export default AudioService
\ No newline at end of file
+// must be in this format
+module.exports = async function () {
```

Comment:
```suggestion
export default async function audio() {
```

Reply:
S tímto zápisem si player neporadí. Proto ten komentář.
<img width="151" alt="Screenshot 2021-11-19 at 10 11 42" src="https://user-images.githubusercontent.com/1457377/142596639-0c6d0bfb-9cfc-46c6-8cc9-18c6cd715499.png">

Reply:
Hmm interesting... odkud se to vola?

Reply:
`trackPlayer.ts`

```
public registerService() {
   RNTrackPlayer.registerPlaybackService(() => require('./services/audio'))
}
```

https://react-native-track-player.js.org/getting-started/

Reply:
Aha tam by musel být normální import, aby to tak fungovalo...

oukej můžem to nechat jak to je